gatsby-browser.js

```diff
@@ -131,15 +133,30 @@ export const onInitialClientRender = () => {
     }
 }
 
+const isStaging = process.env.GATSBY_ENV === 'staging';
+let dataDogVersion = '';
```

Comment:
```suggestion
let dataDogVersion = `deriv-com-${process.env.GIT_TAG_NAME}`;
```

Reply:
We don't need empty variables, let's go with setting up them here and use in the config as variable

Reply:
Same for all of them

Reply:
Okay..I will change it..Thanks